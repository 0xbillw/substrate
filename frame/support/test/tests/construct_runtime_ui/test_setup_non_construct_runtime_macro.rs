#[frame_support::setup_default_test_parameters]
parameter_types! {
    pub const Foo: u32 = 1337;
}

fn main() {}
