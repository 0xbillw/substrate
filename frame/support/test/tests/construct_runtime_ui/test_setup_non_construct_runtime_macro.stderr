error: Invalid test setup attribute macro: expected `construct_runtime!` macro call
 --> $DIR/test_setup_non_construct_runtime_macro.rs:2:1
  |
2 | parameter_types! {
  | ^^^^^^^^^^^^^^^
